packages/code-du-travail-data/dataset/datafiller/fetch.js

```diff
@@ -22,48 +23,50 @@ const getSource = url => {
 
 // sort datafiller references by relevance and source
 const sortRefs = (a, b) => {
+  // 1st sort by relevance
   if (a.relevance < b.relevance) {
-    return -1;
+    return 1;
   } else if (a.relevance > b.relevance) {
+    return -1;
+  }
+  // 2nd sort by sourcesPriority
+  if (
+    sourcesPriority.indexOf(getSource(a.url)) <
+    sourcesPriority.indexOf(getSource(b.url))
+  ) {
+    return -1;
+  } else if (
+    sourcesPriority.indexOf(getSource(a.url)) >
+    sourcesPriority.indexOf(getSource(b.url))
+  ) {
     return 1;
-  } else {
-    if (
-      sourcesPriority.indexOf(getSource(a.url)) <
-      sourcesPriority.indexOf(getSource(b.url))
-    ) {
-      return 1;
-    } else if (
-      sourcesPriority.indexOf(getSource(a.url)) >
-      sourcesPriority.indexOf(getSource(b.url))
-    ) {
-      return -1;
-    }
   }
   return 0;
 };
 
+const hasUrl = row => !!row.url;
+
+const getVariants = row => {
+  const others =
+    (row.variants && row.variants.split("\n").map(variant => variant.trim())) ||
+    [];
+  const variants = [row.title.replace("-", " ")].concat(others);
+  return variants;
```

Comment:
should you return a `[...new Set(variants)]` to get only unique values ?

Reply:
yep, why not :)